src/iotjs_binding_helper.c

```diff
@@ -28,24 +28,24 @@ void iotjs_uncaught_exception(iotjs_jval_t jexception) {
   IOTJS_ASSERT(jerry_value_is_function(jonuncaughtexception));
 
   iotjs_jargs_t args = iotjs_jargs_create(1);
+  jerry_value_clear_error_flag(&jexception);
```

Comment:
I guessed `jexception` should be released somewhere when this function is invoked like `iotjs_uncaught_exception(jerry_acquire_value(jres));` Can you clarify my query?

Reply:
No, `jerry_value_clear_error_flag` will release it and gives a new value.

Reply:
The api reference, https://github.com/jerryscript-project/jerryscript/blob/master/docs/02.API-REFERENCE.md#jerry_value_clear_error_flag, says it clears the error flag. And the example releases the value after calling `jerry_value_clear_error_flag`. Can you kindly tell me what I am still misunderstanding?

Reply:
Maybe we could add more description to the help. The idea is we provide one reference by releasing the old reference and creating a new one. From user perspective: "I still have one reference, just don't use the old value from now on".

Reply:
@daeyeon we investigated this and found a bug in JerryScript. Let's hold this PR until it is fixed.

Reply:
Fixed.